src/AddPopup.js

```diff
@@ -14,9 +14,9 @@ const Popup = ({show, onClose, onSubmit}) => {
     }, [value, onSubmit, onClose]);
 
     return(
-        <div className={'popup' + (show ? '' : ' hidden')} id="myPopUp" >
-            <div className='popupInner'>
-                <h1>Add an Idea</h1>
+        <div className={"popup" + (show ? "" : " hidden")} id="myPopUp" >
+            <div className="popupInner">
```

Comment:
CSS selector of the day: direct descendant.

If you put a ` > ` between two selectors, it creates a selector that only targets the right side if it is a direct child of an element that is targeted by the left side. (This is different from just a space which will target the right side if it's anywhere under the tree of an element targeted by the left side.)

This means that instead of creating another class for "popupInner", you can instead select it with the selector `.popup > div`.